code/controllers/subsystems/job.dm

```diff
@@ -409,9 +409,6 @@
 			var/obj/item/clothing/glasses/G = H.glasses
 			G.prescription = TRUE
 
-	if(H.species)
-		H.species.equip_later_gear(H)
```

Comment:
Uuuh. Is this never being called now, or? (You should start describing your fixes in the PR, so I don't have to ask these questions.)

Reply:
It's called in a different proc. This meant it was called twice, iirc.

Reply:
Line 550 in this file after the changes.